src/coreclr/src/vm/ClrEtwAll.man

```diff
@@ -3714,6 +3727,16 @@
                            task="GarbageCollection"
                            symbol="GCGlobalHeapHistory_V3" message="$(string.RuntimePublisher.GCGlobalHeap_V3EventMessage)"/>
 
+                    <event value="206" version="0" level="win:Informational"  template="GenAwareTemplate"
+                           keywords ="GCHeapDumpKeyword"  opcode="GenAwareBegin"
+                           task="GarbageCollection"
+                           symbol="GenAwareBegin" message="$(string.RuntimePublisher.GenAwareBeginEventMessage)"/>
+
```

Comment:
what did you think of my suggestion of adding the GC index to this event just to make it easier to correlate if you also say have a GCCollectOnly trace going on?